src/app/public/modules/flyout/types/flyout-permalink.ts

```diff
@@ -3,10 +3,24 @@ import {
 } from '@angular/router';
 
 export interface SkyFlyoutPermalink {
+
+  /**
+   * Specifies a text label for the permalink button.
+   */
   label?: string;
+
+  /**
+   * Specifies an object that represents the Angular application route.
+   * The object includes two properties that are mapped to
+   * Angular's [`Router.navigate(commands, extras?)`](https://angular.io/api/router/Router#navigate) method.
```

Comment:
This code snippet is not underscored as a link, so unless users scroll over it, they won't know it's there. I don't think it's worth a `docs-tools` issue since this is probably a fringe case, but what do you think about changing the location of the link? I'm thinking this would be more effective if we moved the links to `Angular application route` in the previous sentence. What do you think?